### DISCUSSION TITLE
How do you make tailwind work on SSR

### DISCUSSION THREAD

**Discussion by elianacris** [vercel/next.js#68172] [Help] [answered]
### Summary

next makes tailwind available as a default style, how do I enable it server-side? 
I have a project in Next and when I disable Js in the browser the tailwind style doesn't work

`import type { Config } from "tailwindcss";

const config: Config = {
  content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
  theme: {
    extend: {
      colors: {
        brand: {
          primary: "rgba(248, 195, 94, 1)",
          secondary: "rgba(248, 195, 94, .8)",
          lighter: "rgba(248, 195, 94, .6)",
          light: "rgba(248, 195, 94, .4)",
          lightest: "rgba(248, 195, 94, .1)",
          "custom-gray": "#ADADAD",
          "custom-light-gray": "#D9D9D9",
          "custom-gray-dark": "#696766",
          yellow: {
            darker: "#A05E03",
            dark: "#FFB323",
            lighter: "#FFC462",
            light: "#FFD188",
            lightest: "#FFEFD7",
          },
          black: {
            primary: "#000000",
            secondary: "rgba(13, 16, 16, .8)",
            lighter: "rgba(13, 16, 16, .6)",
            light: "rgba(13, 16, 16, .4)",
            lightest: "rgba(13, 16, 16, .1)",
          },
          "primary-gray": {
            primary: "rgba(233, 233, 233, 1)",
            base: "rgba(233, 233, 233, 1)",
            lighter: "rgba(233, 233, 233, .6)",
            light: "rgba(233, 233, 233, .4)",
            lightest: "rgba(233, 233, 233, .2)",
          },
          "secondary-gray": {
            primary: "rgba(229, 229, 229, 1)",
            base: "rgba(229, 229, 229, 1)",
            lighter: "rgba(229, 229, 229, .6)",
            light: "rgba(229, 229, 229, .4)",
            lightest: "rgba(229, 229, 229, .2)",
          },
          red: {
            darker: "#D3180C",
            dark: "#FF5247",
            lighter: "#FF6D6D",
            light: "#FF9898",
            lightest: "#FFE5E5",
          },
          green: {
            darker: "#198155",
            dark: "#23C16B",
            lighter: "#4CD471",
            light: "#7DDE86",
            lightest: "#ECFCE5",
          },
          blue: {
            darker: "#0065D0",
            dark: "#48A7F8",
            lighter: "#6EC2FB",
            light: "#9BDCFD",
            lightest: "#C9F0FF",
          },
        },
        red: {
          darker: "#D3180C",
          dark: "#FF5247",
          lighter: "#FF6D6D",
          light: "#FF9898",
          lightest: "#FFE5E5",
        },
        green: {
          darker: "#198155",
          dark: "#23C16B",
          lighter: "#4CD471",
          light: "#7DDE86",
          lightest: "#ECFCE5",
        },
        blue: {
          darker: "#0065D0",
          dark: "#48A7F8",
          lighter: "#6EC2FB",
          light: "#9BDCFD",
          lightest: "#C9F0FF",
        },
      },
      fontFamily: {
        sans: ["var(--font-inter)"],
        inter: ["Inter", "Manrope", "Poppins", "sans-serif"],
        poppins: ["Poppins", "Inter", "Manrope", "sans-serif"],
        manrope: ["Manrope", "Poppins", "Inter", "sans-serif"],
      },
      fontSize: {
        title1: "4rem",
        title2: "2.625rem",
        title3: "2rem",
        large: "1.5rem",
        regular: "1.25rem",
        small: "1.125rem",
        tiny: "1rem",
      },
    },
  },
  plugins: [require("autoprefixer")],
  corePlugins: {
    preflight: false,
  },
};
export default config;`

**Comment by icyJoseph** [comment, marked as answer]
Did you test this on Dev Mode? Dev Mode uses a bit of JavaScript runtime to delivery tailwind styles. However, if you `next build`, and then `next start`, even with JS disabled, you should be able to see the proper styles on the page.

**Reply by elianacris** [reply]
Your suggestion worked, thank you
